src/patternfly/components/Icon/icon.scss

```diff
@@ -0,0 +1,134 @@
+.pf-c-icon {
+  // Example block variable
+  --pf-c-icon--Width: var(--pf-global--icon--FontSize--md);
+  --pf-c-icon--Height: var(--pf-global--icon--FontSize--md);
+
+  // Size modifiers
+  --pf-c-icon--m-sm--Width: var(--pf-global--icon--FontSize--sm);
+  --pf-c-icon--m-md--Width: var(--pf-global--icon--FontSize--md);
+  --pf-c-icon--m-lg--Width: var(--pf-global--icon--FontSize--lg);
+  --pf-c-icon--m-xl--Width: var(--pf-global--icon--FontSize--xl);
+  --pf-c-icon--m-sm--Height: var(--pf-global--icon--FontSize--sm);
+  --pf-c-icon--m-md--Height: var(--pf-global--icon--FontSize--md);
+  --pf-c-icon--m-lg--Height: var(--pf-global--icon--FontSize--lg);
+  --pf-c-icon--m-xl--Height: var(--pf-global--icon--FontSize--xl);
+
+  // Content variables
+  --pf-c-icon__content--Color: var(--pf-global--icon--Color--dark);
```

Comment:
Hmm... this isn't as friendly with dark theme. IMO you could set this to the default text color, which is something I think we'll update with the global var refresh. Most (all?) non-status icons are either the primary text color (just an icon) or secondary text color (when used on its own as an interactive thing, like an action), and it updates to the default text color when interacted with.

The plain button is an example of that:

https://github.com/patternfly/patternfly/blob/9ab78a81b9d6d63cd7795fa82b1ff673abba5332/src/patternfly/components/Button/button.scss#L126-L136

Reply:
I think we discussed elsewhere that this variable actually does work ok in dark theme, and I've also refactored the inline variant to use the text color.